src/apps/loyalty/server/index.tsx

```diff
@@ -71,4 +72,18 @@ app.get("/inquiries", (req, res) => {
   }))
 })
 
+app.get("/thankyou", (req, res) => {
+  if (!req.user) {
+    return res.redirect(req.baseUrl + "/login")
+  }
+
+  const html = renderToString(<Thankyou />)
+  const styles = styleSheet.rules().map(rule => rule.cssText).join("\n")
+  res.send(renderPage({
+    styles,
+    html,
+    entrypoint: "/bundles/thankyou.js",
+  }))
```

Comment:
I actually didn't think as deeply as you did and just copy&pasted an existing one. And yes, client-side javascript is unnecessary on this page. Maybe the entrypoint could be taken out, I'll look into how we can simplify it.